playbooks/installer/ospd/overcloud/templates/overcloud_deploy.sh.j2

```diff
@@ -8,7 +8,7 @@ openstack overcloud deploy --debug \
                        --neutron-network-type {{ installer.overcloud.network.backend }} \
 {% if installer.overcloud.network.backend == "vlan" %}
                        --neutron-disable-tunneling \
-                       --neutron-bridge-mappings {{ installer.undercloud.config.physical_network }}:br-ex,tenant:br-isolated \
+                       --neutron-bridge-mappings {{ installer.undercloud.network.physical_network }}:br-ex,tenant:br-isolated \
```

Comment:
This _**should**_ be 'installer.undercloud.config.physical_network' unless you changed the undercloud config itself under settings/installer/ospd.yml